editor/src/components/canvas/canvas-strategies/absolute-resize-bounding-box-strategy.tsx

```diff
@@ -149,7 +150,8 @@ export const absoluteResizeBoundingBoxStrategy: CanvasStrategy = {
                   elementParentBounds,
                   edgePosition,
                 ),
-                setSnappingGuidelines('transient', guidelinesWithSnappingVector),
+                setSnappingGuidelines('transient', guidelinesWithSnappingVector), // TODO I think this will override the previous snapping guidelines
```

Comment:
What's the TODO? :)

Reply:
ah yeah, to investigate / make sure we are not making a mistake here